src/symbolic_utilities.jl

```diff
@@ -80,8 +80,8 @@ function transform_expression(pinnrep::PINNRepresentation, ex; is_integral = fal
     return ex
 end
 
-function get_ε(dim, der_num, eltypeθ)
-    epsilon = cbrt(eps(eltypeθ))
+function get_ε(dim, der_num, eltypeθ, order)
+    epsilon = ^(eps(eltypeθ), one(eltypeθ) / (2 + order))
```

Comment:
This seems off by one? It should be `sqrt(eps)` at first order.

Reply:
It was cbrt(eps). When order = 1 they are consistent.

Reply:
oh yes, double checked the formulae. It's +2 for central differences like in https://github.com/JuliaDiff/FiniteDiff.jl/blob/master/src/epsilons.jl#L30